src/components/LineChart/Line-view.test.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import { shallow } from 'enzyme';
-import LineChart from './Line';
+import LineChart from './Line-view';
```

Comment:
Confusing, should be `LineView` instead or change the filename to `LineChart-view`